packages/relay/src/lib/services/ethService/ethFilterService/index.ts

```diff
@@ -118,6 +118,19 @@ export class FilterService implements IFilterService {
     }
   }
 
+  async newBlockFilter(requestIdPrefix?: string): Promise<string | JsonRpcError> {
+    this.logger.trace(`${requestIdPrefix} newBlockFilter()`);
+    try {
+      FilterService.requireFiltersEnabled();
+      return this.createFilter(constants.FILTER.TYPE.NEW_BLOCK, {
```

Comment:
Q: do handle if a matching filter already exists or would that end up extending the TTL of the existing filter?

Reply:
Calling `eth_newFilter` or `eth_newBlockFilter` always creates a new filter with a unique `filterId`. Every filter has its own cached `lastQueried` value, which is going to be used as a `fromBlock` when `eth_getFilterChanges` is called for that filter. So when 2 filters are created with `eth_newBlockFilter`, they will return different results depending on the last time they were called.

The TTL is going to be extended on `eth_getFilterChanges` and `eth_filterLogs`